include/ofxPixels.h

```diff
@@ -78,7 +78,7 @@ typedef struct OfxRGBColourF {
 
 /** @brief Defines a double precision floating point component RGB pixel */
 typedef struct OfxRGBColourD {
-  double r, g, b, a;
+  double r, g, b;
```

Comment:
actually, libSupport had its own definition in ofxsCore.h:
```
Should migrate this to the ofxCore.h in a v1.1
*/
struct OfxRGBColourD {
  double r, g, b;
};
```
I'm preparing a pull request for Support, which includes fixing that too